src/Core/Framework/Test/TestCaseBase/SystemConfigTestBehaviour.php

```diff
@@ -2,7 +2,8 @@
 
```

Comment:
I am not sure if this trait is actually needed anymore as `ensureKernelShutdown` would trigger the `reset` method of the service. However, I am not sure the tests call ensureKernelShutdown in all relevant scenarios.

Reply:
I could, however, also retrieve the service_resetter service and reset all services that way instead but then the trait name is kind of wrong.